### PR TITLE
Fixes by niteo

### PR DESCRIPTION
**Pull request by NorfairKing**
@adisbladis As discussed, here are the overrides that we have had to use.

This work has been sponsored by [niteoweb](https://github.com/niteoweb/)

### CODE REVIEW

**Comment by NorfairKing** [comment]
I cannot tell whether the CI failure has anything to do with this PR because CI was already broken on master... What do you think @adisbladis ?

**Comment by adisbladis** [comment]
@NorfairKing The CI breakage is an unrelated nixpkgs regression.

**Comment by NorfairKing** [comment]
@adisbladis Alright then! Is there anything else that I need to do to move this forward?

File: overrides.nix [hunk 1]
```diff
@@ -985,6 +985,11 @@ self: super:
   pytest = super.pytest.overridePythonAttrs (
     old: {
       doCheck = false;
+
+      # Fixes https://github.com/pytest-dev/pytest/issues/7891
+      postPatch = old.postPatch or "" + ''
+        sed -i '/\[metadata\]/aversion = ${old.version}' setup.cfg
+      '';
```
Comment (Infinisil) [inline overrides.nix:992]: Can be removed now that #217 is merged

**Comment by NorfairKing** [comment]
@Infinisil I took out that commit and rebased onto master.

**Comment by NorfairKing** [comment]
Thanks @adisbladis !

**Pull request merged**